mu-plugins/blocks/query-filter/postcss/style.pcss

```diff
@@ -0,0 +1,314 @@
+	padding: 0;
+	position: absolute;
+	width: 1px;
+	word-wrap: normal !important;
+
+	& + label {
+		display: block;
+		margin: 0;
+		padding-block: var(--wp--custom--wporg-query-filters--spacing--padding--block);
+		padding-left: var(--wp--custom--wporg-query-filters--spacing--padding--inline-start);
+		padding-right: calc(var(--wp--custom--wporg-query-filters--spacing--padding--inline-end) * 2 + var(--wporg-query-filter--icon-size));
+	}
+
+	&:focus + label {
+		outline: 1.5px solid var(--wp--custom--wporg-query-filters--toggle--focus--color--outline);
+		outline-offset: -1.5px;
+		box-shadow: inset 0 0 0 3px var(--wp--custom--wporg-query-filters--toggle--focus--color--inset);
+	}
+
+	&:checked + label {
+		border-radius: 2px;
+		background-color: var(--wp--custom--wporg-query-filters--option--active--color--background);
+		color: var(--wp--custom--wporg-query-filters--option--active--color--text);
+		/* stylelint-disable-next-line function-url-quotes */
+		background-image: url("data:image/svg+xml,%3Csvg width='24' height='24' viewBox='0 0 24 24' fill='none' xmlns='http://www.w3.org/2000/svg'%3E%3Cpath d='M12 13.0607L15.7123 16.773L16.773 15.7123L13.0607 12L16.773 8.28772L15.7123 7.22706L12 10.9394L8.28771 7.22705L7.22705 8.28771L10.9394 12L7.22706 15.7123L8.28772 16.773L12 13.0607Z' fill='%231E1E1E'/%3E%3C/svg%3E%0A");
+		background-repeat: no-repeat;
+		background-position: right var(--wp--custom--wporg-query-filters--spacing--padding--inline-end) center;
+	}
+}
+
+.wporg-query-filter__modal-actions {
+	display: flex;
+	justify-content: space-between;
+	padding: var(--wp--preset--spacing--10);
+	border-top:
+		var(--wp--custom--wporg-query-filters--border--width)
+		var(--wp--custom--wporg-query-filters--border--style)
+		var(--wp--custom--wporg-query-filters--border--color);
+}
+
+.wporg-query-filter__modal-actions input {
+	padding: 12px;
+	font-weight: var(--wp--custom--button--typography--font-weight);
+	font-size: var(--wp--custom--button--typography--font-size);
+	line-height: var(--wp--custom--button--typography--line-height);
+	color: var(--wp--custom--button--color--text);
+	background-color: var(--wp--custom--button--color--background);
+	border: none;
+	border-radius: 2px;
+
+	&:hover,
+	&:focus {
+		color: var(--wp--custom--button--hover--color--text);
+		background-color: var(--wp--custom--button--hover--color--background);
+	}
+
+	&:focus {
+		background-color: var(--wp--custom--button--color--background);
+		box-shadow: inset 0 0 0 3px var(--wp--preset--color--white);
+		outline: 1.5px solid var(--wp--custom--button--focus--border--color);
+		outline-offset: -1.5px;
+	}
+
+	&:active {
+		color: var(--wp--custom--button--active--color--text);
+		background-color: var(--wp--custom--button--active--color--background);
+	}
+
+	&.wporg-query-filter__modal-action-clear {
+		background-color: transparent;
+		color: #a13f2a; /* pomegrade-1 fails contrast against white. */
+		font-weight: 400;
+
+		&:hover,
+		&:active {
```

Comment:
Should this have a `:focus` state?

Reply:
It inherits the focus from the focus just above here, it doesn't need a custom focus style— but if design wants to create one I can change it.

<img width="394" alt="Screenshot 2023-09-06 at 11 01 10 AM" src="https://github.com/WordPress/wporg-mu-plugins/assets/541093/65cab3c8-0222-45e6-adfc-ccdfea7ea7df">